fetch-metadata/src/util/MetaData.ts

```diff
@@ -0,0 +1,11 @@
+import {MetaManifest} from "@web3api/client-js";
+
+export interface MetaData extends MetaManifest {
+  iconImage?: string;
+  links?: {
```

Comment:
Why are we doing this? Shouldn't we just use the `MetaManifest` type directly, without these added fields?

Reply:
The `icon` and `path` properties in the meta manifest have paths from the wrapper root to the image files. The iconImage properties contain base64 string representations of the image data, which is fetched from the paths. Organizing it in the same object made it easier to map over the links array and display the images.

Reply:
I refactored to use the `MetaManifest` type directly